### PR TITLE
Tests fixes.

### PR DESCRIPTION
**Pull request by hjafarpour**
The updates from the previous merge was not applied to the following two classes. This PR fixes it.

### CODE REVIEW

**Comment by hjafarpour** [comment]
This is not in 4.1.x and only in master. Doesn't seem to be a blocker.

**Comment by apurvam** [comment]
Ah. So it is a follow up PR, not a fix for a bad merge?

**Comment by hjafarpour** [comment]
Yes, follow up PR although it could also be included in the 4.1.x merge.

File: ksql-engine/src/test/java/io/confluent/ksql/util/FakeKafkaTopicClient.java [hunk 1]
```diff
@@ -40,12 +40,12 @@
     final String topicName;
     final int numPartitions;
     final short replicatonFactor;
-    final String cleanupPolicy;
+    final TopicCleanupPolicy cleanupPolicy;
```
Comment (big-andy-coates) [inline ksql-engine/src/test/java/io/confluent/ksql/util/FakeKafkaTopicClient.java]: should probably make these `private`.

**Pull request merged**